.github/workflows/quarkus-nightly.yml

```diff
@@ -0,0 +1,78 @@
+name: Quarkus Nightly
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  push:
+    branches:
+      - quarkus-nightly
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+concurrency:
+  # Only cancel jobs for PR updates
+  group: quarkus-nightly-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  rebase-quarkus-nightly-branch:
+    name: Rebase quarkus-nightly branch
```

Comment:
I'd name the branch `quarkus-next` instead of nightly, as we would also want this branch to serve as a placeholder for any commits needed to get ready for the next release.

Reply:
Addressed.